sdk/ml/azure-ai-ml/tests/test_configs/internal/distribution-component/component_spec.yaml

```diff
@@ -22,5 +22,5 @@ launcher:
     python train.py --input-path {inputs.input_path} [--string-parameter {inputs.string_parameter}]
     --output-path {outputs.output_path}
 environment:
-  name: AzureML-Minimal
```

Comment:
why this?

Reply:
[exp link](https://ml.azure.com/experiments/id/b1eff82f-aeac-434f-b327-1435eb9cfa9e/runs/eager_rose_j290z9x280?wsid=/subscriptions/74eccef0-4b8d-4f83-b5f9-fa100d155b22/resourcegroups/AmlComponentNotebook/providers/Microsoft.MachineLearningServices/workspaces/SDK-Notebook-Stable-EUS-20230529&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

Reply:
ignore me, this spec is not used in our current samples. Will update the spec in further PR.